src/services/codefixes/importFixes.ts

```diff
@@ -959,7 +937,7 @@ function getNewImportFixes(
     let computedWithoutCacheCount = 0;
     const fixes = flatMap(exportInfo, (exportInfo, i) => {
         const checker = getChecker(exportInfo.isFromPackageJson);
-        const { computedWithoutCache, moduleSpecifiers } = getModuleSpecifiers(exportInfo.moduleSymbol, checker);
+        const { computedWithoutCache, moduleSpecifiers } = targetFileImportFrom ? { computedWithoutCache: true, moduleSpecifiers: targetFileImportFrom } : getModuleSpecifiers(exportInfo.moduleSymbol, checker);

```

Comment:
This `targetFileImportFrom` seems weird to me. What would it take to compute the module specifier here instead of in `moveToFile`?

Reply:
For tests like `tests/cases/fourslash/moveToFile_existingImports5.ts` where a symbol exists in `b.ts` and is being moved to a different file, say  `a.ts`. There is a possibility that it needs to be imported by `b.ts`. This is where I was using the `targetFileImportFrom` to replace the module specifier with `a.ts` to create something like `import { Foo } from './a'`.

In `addImportFromSymbol()`, for the case where a symbol doesn't have a module specifier, I assign `modulesFileName` to  `targetFilePath`, which is `a.ts` and I can do something similar for the else case in there too to the `exportInfo`. Then in `getNewImportFixes()` I can compare if `moduleSpecifiers !== exportInfo.moduleFileName` then replace `moduleSpecifiers` with the `moduleFileName`.
Would that look better?